crates/ruff_linter/src/rules/flake8_simplify/rules/enumerate_for_loop.rs

```diff
@@ -11,6 +13,8 @@ use crate::checkers::ast::Checker;
 /// Checks for `for` loops with explicit loop-index variables that can be replaced
 /// with `enumerate()`.
 ///
+/// In [preview], this rule checks for index variable initialized as an instance of `int`.
```

Comment:
```suggestion
/// In [preview], this rule checks for index variables initialized with any integer rather than only
/// a literal zero.
```

Having to say this is making me rethink the preview suggestion 😅 But I guess it's better to be safe. Thanks for updating the docs!